libs/design-system/src/lib/Components/LandingHeader/LandingHeaderContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import clsx from 'clsx';
+import React from 'react';
+
+export type GradientName =
+  | 'community'
+  | 'concepts'
+  | 'getting-started'
+  | 'home'
+  | 'network'
+  | 'tools';
+
+export type LandingHeaderContainerProps =
+  React.ComponentPropsWithoutRef<'header'> & {
+    gradient: GradientName;
+  };
+
+export function LandingHeaderContainer({
+  className,
+  gradient,
+  ...props
+}: LandingHeaderContainerProps) {
+  return (
+    <header
+      className={clsx(`bg-gradient-${gradient} bg-cover`, className)}
```

Comment:
Nice, I think background image is the way to go for these. I think the headers could have more space at the top, and the gradients could move up higher to match the pages in the Flow Dev Portal figma